controller/oidc_auth/storage.go

```diff
@@ -91,7 +91,7 @@ type HybridStorage struct {
 	serviceUsers cmap.ConcurrentMap[string, *Client]
 
 	startOnce sync.Once
```

Comment:
If this is unused, should we just remove it?

Reply:
dunno. it's an @andrewpmartinez question. I just made the linter happy :) i was pointed it out to andrew...